File: Gerenciador de Plano de Aula/Web.Server/Models/Usuario.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Web.Server.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string Cpf { get; set; }
        public DateTime DataNascimento { get; set; }
        public string Email { get; set; }
        public Cargos Cargo { get; set; }
        public string Senha { get; set; }

        [NotMapped]
        public string Token { get; set; }
    }
}

File: Gerenciador de Plano de Aula/Web/ClientApp/src/app/models/usuario.ts
export class Usuario {
  id: string;
  nome: string;
  sobrenome: string;
  dataNascimento: Date;
  email: string;
  senha: string;
  cargo: string;
  token: string

  get nomeCompleto(): string {
    return `${this.nome} ${this.sobrenome}`;
  }

  static usuarioMap(usuario: Usuario): Usuario {
    let u = new Usuario();

    u.id = usuario.id;
    u.nome = usuario.nome;
    u.sobrenome = usuario.sobrenome;
    u.email = usuario.email;
    u.senha = usuario.senha;
    u.cargo = usuario.cargo;
    u.token = usuario.token;

    return u;
  }
}
